jekyll/_cci2/plan-free.adoc

```diff
@@ -16,7 +16,7 @@ This document describes the Free plan available to developers.
 
 [#free-plan]
 == Free plan
-As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient, and fast.
+CircleCI supports a free-tier with the usage-based plan. By using the Free plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient, and fast.
```

Comment:
I think this part can be simplified because there are _only_ usage based plans now. Something like "Use the Free plan to take advantage of ...." maybe?